modules/transactions/src/testFixtures/java/org/apache/ignite/internal/tx/test/TestTransactionIds.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.tx.test;
+
+import java.util.UUID;
+import org.apache.ignite.internal.hlc.HybridClock;
+import org.apache.ignite.internal.hlc.HybridClockImpl;
+import org.apache.ignite.internal.tx.TransactionIds;
+
+/**
+ * Contains methods to generate transaction IDs for the cases when the clock and nodeId might be hard-coded
+ * (just for tests).
+ */
+public class TestTransactionIds {
+    /** Global clock instance. */
+    private static final HybridClock GLOBAL_CLOCK = new HybridClockImpl();
+
+    /** Hard-coded node ID. */
+    private static final int SOLE_NODE_ID = 0xdeadbeef;
```

Comment:
Can you use `TransactionIdGenerator` here? This class feels like a copy of it

Reply:
Not sure if this is what you ment, let's see. The thing is that `TransactionIdGenerator` only encapsulates nodeId, not the clock. Maybe not the best name for the abstraction.